packages/flutter/lib/src/material/floating_action_button_theme.dart

```diff
@@ -129,6 +132,11 @@ class FloatingActionButtonThemeData with Diagnosticable {
   /// The text style for an extended [FloatingActionButton]'s label.
   final TextStyle? extendedTextStyle;
 
+  /// {@macro flutter.material.RawMaterialButton.mouseCursor}
+  ///
+  /// If specified, overrides the default value of [FloatingActionButton.mouseCursor].
```

Comment:
should this be worded in a better way? ik this is how it's explained everywhere else, but even i got confused as to what will this override.